### PR TITLE
一回も未ログイン時に商品を追加しないとログイン時にaccessIdがない現象に対応

### PR DESCRIPTION
**Pull request by tkeshun**

### CODE REVIEW

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 1]
```diff
@@ -60,8 +60,10 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
 			accessId = user.getId();
 			System.out.println("update:id" + user.getId());
 			System.out.println("showカーと時のsession: " + httpSession.getId().hashCode());
-			String sessionId = httpSession.getAttribute("accessId").toString();
-			cartService.updateUserId(accessId, Integer.parseInt(sessionId));
+			Object sessionId = httpSession.getAttribute("accessId");
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:63]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:63]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'else' child has incorrect indentation level 24, expected level should be 6.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 2]
```diff
@@ -60,8 +60,10 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
+			if (sessionId != null) {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:64]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:64]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' has incorrect indentation level 24, expected level should be 6.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 3]
```diff
@@ -60,8 +60,10 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
+				cartService.updateUserId(accessId, Integer.parseInt(sessionId.toString()));
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 107).

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 32, expected level should be 8.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 4]
```diff
@@ -60,8 +60,10 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
+			}
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:66]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:66]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if rcurly' has incorrect indentation level 24, expected level should be 6.

**Review by yousuke-murayama — APPROVED** [review]
問題ないと思います。

**Pull request merged**